spec/rubocop/cop/lint/uri_regexp_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::Lint::UriRegexp do
+  let(:config) { RuboCop::Config.new }
+  subject(:cop) { described_class.new(config) }
+
+  it 'registers an offense when using `URI.regexp` with argument' do
+    expect_offense(<<-RUBY.strip_indent)
+      URI.regexp('http://example.com')
```

Comment:
What do you think about having the offense highlight more of the expression? It seems like this should highlight `URI.regexp` or `URI.regexp('http://example.com')`

Reply:
I do not know if that is correct, but since `URI` module does not change, this cop focused on only `regexp`.